requirements-build.in

```diff
@@ -1,4 +1,4 @@
-Cython
+Cython<3.0
```

Comment:
did you add this one manually? I'm afraid we will need to force it on makefile (like setuptools< 67 was)

Reply:
I'd add it in the Makefile section generating build-requirements.in.
But since the normal process fixed it, lets leave it as is.